lib/rollbar/tasks/rollbar.cap

```diff
@@ -31,6 +31,15 @@ namespace :rollbar do
 end
 
 namespace :deploy do
+  task :get_current_revision do
+    on primary fetch(:rollbar_role) do
+      within release_path do
+        set(:current_revision, capture(:cat, 'REVISIONS'))
```

Comment:
Should probably also redirect STDERR to /dev/null so that current_revision doesn't get set if the REVISIONS file is missing for some reason.  Also, isn't the file named 'REVISION' and not 'REVISIONS'?

```ruby
set(:current_revision, capture(:cat, 'REVISION', '2>/dev/null'))
```

My knowledge of Capistrano is a bit lacking, does this file always exist or is it only for Capistrano 3+?